definitions/fenix.toml

```diff
@@ -371,3 +378,9 @@ from_expression = """(
 experiments_column_type = "simple"
 client_id_column = "client_id"
 submission_date_column = "submission_date"
+
+[data_sources.new_profile_activation]
+from_expression = "`moz-fx-data-shared-prod.fenix.new_profile_activation`"
+experiments_column_type = "simple"
+client_id_column = "client_id"
+submission_date_column = "submission_date"
```

Comment:
What is the submission date used for? My table is partitioned on submission_date, but for most reporting it'd be ideal to use first_seen_date, which is lagged by 7 days.

Reply:
Submission date is used for metrics in order to determine which records should be counted, mozanalysis [looks for rows ](https://github.com/mozilla/mozanalysis/blob/main/src/mozanalysis/metrics/__init__.py#L158-L162) where the submission date is within a particular observation window. So for example, for the week-2 results it looks for rows where the submission date is 7 to 14 days after the enrollment date. 

For your new activation metric, does submission date mean one of:
1. The date they activated (could be any time within first week after first seen)
2. Always 7 days after first seen

Option 2 will work as written, option 1 won't

Reply:
Ok, great. It's option 2, it's computed exactly 7 days after.